engine/app/components/citizens_advice_components/input.rb

```diff
@@ -86,10 +90,17 @@ def base_input_attributes
         value: value,
         required: required?,
         "aria-invalid": error?,
-        "aria-describedby": (error_id.to_s if error?)
+        "aria-describedby": described_by
       }
     end
 
+    def described_by
+      classes = []
```

Comment:
💅 Nitpick: These are IDs not class names

Reply:
I'm so embarrassed.